bin/v-restart-proxy

```diff
@@ -70,6 +70,14 @@ if [ -f "$HESTIA/web/inc/nginx_proxy" ]; then
         nohup $BIN/v-restart-proxy 'background' &>/dev/null &
     else
     # Default behaviour
+    
+    # Preform an check if Nginx is valid as reload doesn't throw an error / exit
+    result=$(service $WEB_SYSTEM $PROXY_SYSTEM  >> /dev/null 2>&1 )
```

Comment:
This will always fail

Reply:
You hav a point it runs now service apache2 nginx .. 
result is not being used so it can be removed either 

https://github.com/jaapmarcus/hestiacp/commit/0222ed729dd054fe54210a510fbb776a4bcb2748